rclcpp/include/rclcpp/node.hpp

```diff
@@ -1591,6 +1597,9 @@ class Node : public std::enable_shared_from_this<Node>
   const rclcpp::NodeOptions node_options_;
   const std::string sub_namespace_;
   const std::string effective_namespace_;
+
+  class BackportMemberMaps;
```

Comment:
Note to reviewer: went with a generic name to leave it open for possible other members usage

Reply:
Maybe the intention wasn't clear. This class could have more maps within it, controlled by the same access mutex, that create more "members" for the owning class. Given that members would be created all at once, and removed all at once, there would be no partial construction or partial deletion of these "members" of a class instance, it makes sense to use the same mutex and manage all the maps at once. That was the intention here - this thing is "all extra backported members" - and maybe that ends up being only one, but if we do want to backport more member-based features to Iron we don't need to create any new classes or static members, this one does the whole job via new maps within it. I like it as a generic backport pattern that could be reused.

Reply:
The implementation could actually be a singular map to some "tuple" or struct of extra members, that is added and deleted atomically.

Reply:
Ah, OK.  Thanks for the explanation.  In that case, I'm fine with the name as-is.  I'll just suggest you put some of that explanation there into a comment (either here in the `.hpp` or in the `.cpp`, I don't care which).